Chart.yaml

```diff
@@ -18,12 +18,14 @@ annotations:
     - name: consul-k8s
       image: hashicorp/consul-k8s:0.26.0-beta2
     - name: envoy
-      image: envoyproxy/envoy-alpine:v1.16.0
+      image: envoyproxy/envoy-alpine:v1.18.2
   artifacthub.io/license: MPL-2.0
   artifacthub.io/links: |
-    - name: Consul Homepage
-      url: https://www.consul.io
     - name: Documentation
       url: https://www.consul.io/docs/k8s
-    - name: Chart GitHub Repository
+    - name: hashicorp/consul-helm
```

Comment:
Do we want to remove these from sources above?

Reply:
Artifacthub will do the de-duplication so my thinking was that in case there are other tools that don't use the artifacthub annotations, we should keep the `sources` for those.